api/v3/AwardImport.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * @file
+ * Award Import API file.
+ */
+
+use CRM_CiviAwards_Service_AwardImport as AwardImportService;
+
+/**
+ * AwardImport.create API specification.
+ *
+ * @param array $spec
+ *   Description of fields supported by this API call.
+ */
+function _civicrm_api3_award_import_create_spec(array &$spec) {
+  $spec['title'] = [
+    'type' => CRM_Utils_Type::T_STRING,
+    'title' => ts('Award Title'),
+    'description' => ts('Natural language name for the Award'),
+  ];
+
+  $spec['description'] = [
+    'type' => CRM_Utils_Type::T_STRING,
+    'title' => ts('Description'),
+    'description' => ts('Description of the Award'),
+  ];
+
+  $spec['is_active'] = [
+    'type' => CRM_Utils_Type::T_BOOLEAN,
+    'title' => ts('Is Active'),
+    'description' => ts('Is this Award active?'),
+  ];
+
+  $spec['award_subtype'] = [
+    'type' => CRM_Utils_Type::T_STRING,
+    'title' => CRM_CiviAwards_ExtensionUtil::ts('Award Subtype'),
+    'description' => CRM_CiviAwards_ExtensionUtil::ts('One of the values of the award_subtype option group'),
+  ];
+
+  $spec['start_date'] = [
+    'type' => CRM_Utils_Type::T_DATE,
+    'title' => CRM_CiviAwards_ExtensionUtil::ts('Start Date'),
+    'description' => CRM_CiviAwards_ExtensionUtil::ts('Award Start Date'),
+  ];
+
+  $spec['end_date'] = [
+    'type' => CRM_Utils_Type::T_DATE,
+    'title' => CRM_CiviAwards_ExtensionUtil::ts('End Date'),
+    'description' => CRM_CiviAwards_ExtensionUtil::ts('Award End Date'),
+  ];
+
+  $spec['is_template'] = [
+    'type' => CRM_Utils_Type::T_BOOLEAN,
+    'title' => CRM_CiviAwards_ExtensionUtil::ts('Is Template?'),
+    'description' => CRM_CiviAwards_ExtensionUtil::ts('Whether the award detail is for a template or not'),
+  ];
+
+  $spec['award_manager'] = [
+    'type' => CRM_Utils_Type::T_STRING,
+    'title' => 'Award Manager',
+    'description' => 'A comma-separated list of Contact IDs',
+  ];
+}
```

Comment:
Implemented here: https://github.com/compucorp/uk.co.compucorp.civiawards/pull/164/commits/109c30f27319c60f71d4f38c9578b0ca633141bf
This was failing because of, from the `getfields` API call, we got `award_subtype` field with an element named `pseudoconstant`. 
I don't know the exact reason but this was making the import to fail. Other fields also had the `pseudoconstant` key, but in them worked well, then is very specific of this only spec. 
I overwrite the value with the manual creation, without the problematic element.
Let me know if this sounds good.